compiler/erg_parser/parse.rs

```diff
@@ -1469,6 +1449,18 @@ impl Parser {
             Some(t) if t.category_is(TC::Literal) => {
                 // TODO: 10.times ...などメソッド呼び出しもある
```

Comment:
> TODO: 10.times ...などメソッド呼び出しもある
Please tell me about this TODO
I'm going to check as invalid syntax

Reply:
Probably forgot to erase this comment. It is no longer meaningful.